lib/better_errors/version.rb

```diff
@@ -1,3 +1,3 @@
 module BetterErrors
-  VERSION = "2.1.1"
+  VERSION = "2.1.1.1"
```

Comment:
I think you shouldn’t change the VERSION in a pull request. If v2.2.0 is released before this PR gets merged, you’d need to change it.